lib/config.ts

```diff
@@ -0,0 +1,58 @@
+// Inspiration from https://github.com/finnishtransportagency/hassu/blob/main/deployment/lib/config.ts
```

Comment:
Should/can this env config file be used for all packages simultaneously?

Reply:
That is the plan. That's what the `getRataExtraStackConfig` is representing. There's just no values there yet.